app/index.jsx

```diff
@@ -2,8 +2,23 @@ var React = require('react')
 var ReactDOM = require('react-dom')
 const constants = require('./scripts/constants.js')
 
-var Nav = require('./components/nav.jsx')
-var BootstrapPre = require('./components/bootstrap-pre.jsx')
+var Navigation = require('./components/navigation.jsx')
+const componentCode = [
```

Comment:
It's not really. All you'll have to do is move all of the files you created into an elements folder
Then create an elements file in the components folder.
The copy paste the require statements into the elements file.

I can do it if you want, it'll only take a couple of minutes.